### DISCUSSION TITLE
Request: 更新日時を変えずにページを更新したい

### DISCUSSION THREAD

**Discussion by aviscaerulea** [growilabs/growi#7737] [Ideas / Requests]
ページ名の変更ではメタ情報を変更せずに行うことができますが、ページの内容を更新する際も同じように更新日時は維持したまま更新できるようにしたいことがあります。

記事内のちょっとした TYPO などの軽微な修正で更新日時が変わると、更新一覧など上位に出てきてしまうので #7280 でリクエストさせていただた件と併せて、他のページで何が更新されたか分かり難くなってしまいます。

運用として検索してページを探すよりも、ニュース的な更新された新しいページを見るという使い方もしているので、更新一覧が重要になっています。

| item     | version |
| ---      | --- |
|OS        |Amazon Linux2|
|GROWI     |5.0.4|
|node.js   |16.15.0|
|npm       |8.5.5|
|yarn      |1.22.18|
|Using Docker|yes|
|Using [growi-docker-compose][growi-docker-compose]|yes|

**Comment by yuki-takei** [comment]
@aviscaerulea ご意見ありがとうございます。

「更新日時」というデータの在り方として、内容の書き換えが伴う場合は更新すべきかなと思います。

その前提だとすると「更新日時以外のソート」によるリスト表示の手段がないことが問題なのかなという所感で、#7280 の lsx 拡張はアリかなと思います。
サイドバーに出したい場合はメニューの追加や拡張ということになるでしょうか。
